### DISCUSSION TITLE
Help regarding Installation

### DISCUSSION THREAD

**Discussion by PabloArmentano** [OpenAEV-Platform/openaev#1756] [Q&A] [answered]
Hello guys!

I was installing OpenBAS in my local setup for test purposes,  to see how this platform works and I have one question about the installation process.

It wasn't very clear to me reading the manual, but if I choose the manual installation process I already need to have most of the environment installed first (PostgreSQL, RabbitMQ, Caldera etc) to get this platform working?

**Comment by savacano28** [comment, marked as answer]
Hi Pablo!
To run OpenBAS, you'll need to have at least PostgreSQL, Minio, rabbitmq set up and running. These are the minimum requirements to get the application working. https://docs.openbas.io/latest/deployment/installation/#manual-installation

Feel free to reach out in the [Slack community](https://join.slack.com/t/filigran-community/shared_invite/zt-2upf1g4u8-KnduLoFNoqWEMRjgRQa04Q) if you have more questions or need any help!

Thanks,
